mapdata/WarcraftLegacies/BuffData/Skin/1160785986.json

```diff
@@ -6,24 +6,24 @@
   ],
   "Modifications": [
     {
-      "Id": 1952543846,
+      "Id": 1953653094,
       "Type": 3,
-      "Value": ""
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNUnholy.blp"
     },
     {
       "Id": 1885959270,
       "Type": 3,
-      "Value": "Aura of the Reaver"
+      "Value": "Vulnerability Aura"
```

Comment:
The name of this buff should be red because it's negative. Something like "|c00FF0303Vulnerability Aura|r".